Formula/spidermonkey@78.rb

```diff
@@ -0,0 +1,60 @@
+class SpidermonkeyAT78 < Formula
```

Comment:
I still don't understand why we need this. Didn't `couchdb` build with spidermonkey 89?

Reply:
Supported version is hard-coded [here](https://github.com/apache/couchdb/blob/7456125f8ede38ec65cf77d268bc5d953f1f01e1/src/couch/rebar.config.script#L57)